client/src/components/Accordion/Accordion.js

```diff
@@ -119,7 +120,7 @@ const [AdvancedAccess, setAccess] = React.useState(false);
     var safeEmail = "BAD@gmail.com"
   }
   console.log("Email for Register:", safeEmail);
-  axios.get('http://localhost:2222/users', { params: { email: safeEmail } } )
+  httpClient.get(process.env.REACT_APP_API_SERVER + "/users", { params: { email: safeEmail } } )
```

Comment:
Here, `httpClient` is actually a custom wrapper around an axios instance that saves a logged in user's access token for the back-end API (not another httpClient library). I introduced this as part of one of the earlier large PRs, and this probably should have been separated and reviewed as its own PR. The idea is for the client to authenticate just once with the Auth0 API on login, and use the access token it receives for all API calls until it expires in 24 hours, as opposed to getting a new token from the Auth0 tenant for every call made to the backend server.

Reply:
As of this PR, the `/users` endpoint on the server is made accessible only to authenticated users, so changing this line to use our httpClient instead of the default axios instance adds the necessary authorization parameters to this API call.